strongbox-parent/pom.xml

```diff
@@ -92,7 +92,7 @@
         <version.hibernate.validator>6.0.7.Final</version.hibernate.validator>
         <version.glassfish.javax.el>3.0.1-b08</version.glassfish.javax.el>
         <version.antlr>4.7.1</version.antlr>
-        <version.liquibase>3.5.5</version.liquibase>
+        <version.liquibase>3.5.3</version.liquibase>
```

Comment:
@fuss86 : Just for our better understanding (and for future reference) -- what was the reason this had to be rolled back to an older version?